src/main/java/com/adyen/model/checkout/PaymentSessionRequest.java

```diff
@@ -538,6 +534,9 @@ public ApplicationInfo getApplicationInfo() {
     }
 
     public void setApplicationInfo(ApplicationInfo applicationInfo) {
+        if (applicationInfo == null) {
+            throw new IllegalArgumentException("ApplicationInfo cannot be null");
```

Comment:
Should we throw an exception if it is not set? It is not part of required fields

Reply:
We decided to remove the setter, so this is not an issue anymore.